src/components/PieCharts/ProgramSunburst/ProgramSunburstView.js

```diff
@@ -0,0 +1,162 @@
+    }
+    return 0;
+  }
+  return data.caseSize;
+}
+
+const styles = (theme) => ({
+  title: {
+    color: (theme.palette.widgetBackground) ? theme.palette.widgetBackground.contrastText : 'black',
+    fontSize: '12px',
+    maxWidth: '1440px',
+    fontFamily: 'Nunito',
+    lineHeight: '20px',
+    fontWeight: '500',
+    height: '20px',
+    textAlign: 'center',
+  },
+  customWidget: {
+    marginTop: '18px',
+  },
+});
+
+class ProgramSunburst extends PureComponent {
+  constructor(props) {
+    super(props);
+    const { data } = this.props;
+    this.state = {
+      widgetData: data,
+      size: findCaseSizeOfTitle(data, ''),
+      title: '',
+      caseSize: data.children[0].caseSize,
+    };
+  }
+
+  render() {
+    const {
+      caseSize, size, widgetData, title,
+    } = this.state;
+    const {
+      width, height, data, textColor, classes,
+    } = this.props;
+    // update the caseSize  associated with title
+
+    this.setState({
+      widgetData: data,
+      size,
+      title,
+      caseSize: findCaseSizeOfTitle(data, title),
+    });
+
+    return (
+      <>
+        <div className={classes.customWidget}>
+          <Sunburst
+            id={widgetData.key}
+            hideRootNode
+            animation
+            colorType="literal"
+            data={widgetData}
+            height={height}
+            width={width}
+            style={{
+              stroke: '#ddd',
+              strokeOpacity: 0.3,
+              strokeWidth: '0.5',
+            }}
+            onValueMouseOver={(node) => {
+              const path = getKeyPath(node).reverse();
+              const pathAsMap = path.reduce((res, row) => {
+                res[row.toString()] = true;
+                return res;
+              }, {});
+              const wdata = updateData(widgetData, pathAsMap);
+              this.setState({
+                size: node.size,
+                widgetData: wdata,
+                title: node.title,
+                caseSize: node.size || node.caseSize,
+              });
+            }}
+          >
+            {caseSize > 0 && (
+            <LabelSeries data={[{
+              x: 0,
+              y: 0,
+              label: caseSize,
+              style: {
+                fontSize: '12px',
+                textAnchor: 'middle',
+                fill: textColor,
+                fontFamily: '"Nunito","Open Sans", sans-serif',
+              },
+            }, {
+              x: 0,
+              y: 1,
+              label: 'Cases',
+              style: {
+                fontSize: '12px',
+                textAnchor: 'middle',
+                fill: textColor,
+                fontFamily: '"Nunito","Open Sans", sans-serif',
+              },
+            }]}
+            />
+            )}
+          </Sunburst>
+          <div className={classes.title}>
+            {title}
+          </div>
+        </div>
+      </>
+    );
+  }
+}
+
+export default withStyles(styles)(ProgramSunburst);
```

Comment:
Can we use injectSheet from react-jss so we don't need a dependency on MUI for this component